components/parts/Footer.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component /* , PropTypes */ } from 'react';
+
+import * as Firebase from 'functions/firebase';
+
+import Icon from 'parts/Icon';
+import Config from 'Config';
+
+export default class Home extends Component {
+  // static propTypes = {
+  // }
+
+  constructor(props) {
+    super(props);
+    this.ref = Firebase.getFirebaseInstance();
+    this.state = {
+      roomExists: false,
+      roomName: '',
+    };
+  }
+
+  render() {
+    return (
+      <footer>
+        <div className="input-group input-inline">
+          {
+            Config.social.map((link) =>
```

Comment:
Change this line to `Config.social.map((link, index) => .....`

and then change the key to the index. Real likes integer keys.